bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/KeyContent.java

```diff
@@ -12,9 +12,12 @@
  *******************************************************************************/
 package org.eclipse.passage.lic.base.io;
```

Comment:
Please take care of the license header in this file too: 
  - update the _last modified_ date, 
  - include yourself in the list of contricutors.

Reply:
Thanks for pointing out. Done.